Makefile

```diff
@@ -38,6 +38,10 @@ default: fast
 init:
 	mkdir -p dist
 
+.PHONY: generate-cabal-from-hpack
+generate-cabal-from-hpack:
+	stack build --dry-run
```

Comment:
Are you sure you need a make goal for this?  Seems like one of those private tricks that will get lost and just clutter the Makefile.  (I don't insist on removing it, your call.)

Reply:
It's a private trick without which adding a new dependency to package.yaml doesn't work. We seem to have a lot of things but no single source of truth about them, like the docker ephemeral thing.

Reply:
I'll remove this and make a separate PR that gets rid of the package.yaml files.  I remember talking about this in the past, and I think there was something close to consensus about that approach.  Let's see if it passes review!  :)